### DISCUSSION TITLE
Scoring still feels a bit unfair.

### DISCUSSION THREAD

**Discussion by PMorghK** [ppy/osu#17377] [Q&A] [answered]
well I would really like if this gets addressed in the final release of leaderboards and scoring and stuff
![Screenshot from 2022-03-20 19-26-19](https://user-images.githubusercontent.com/43874888/159191148-f50402e0-6783-4f56-b6b1-5288bf8583b4.png)

Like I srsly feel like the A rank I got should be the better score here especially because this is a stamina stream map.

**Comment by frenzibyte** [comment]
Sounds like you're just suggesting an "Accuracy leaderboard"? I can see the first score you have has a much higher combo than the one below it, which would explain why it overtook your "A rank" one.

**Reply by Moltanicaa** [reply]
It also assumes that a combo weight of 70% leads to situations like those described above. It is proposed to increase the weight precisely accuracy.

**Reply by PMorghK** [reply]
Kind of ??? Well im more suggesting that lazer scoring should be more heavily whighted towards acc depending on the map.

Like if im playing quaver (a map with almost all jumps) then it would make sence for the score to be whighted on combo and miss count

But the map I posted is a stamina map and considering that fact I think it should be more heavily whighted on acc. But its not the case here.

**Comment by Methos-** [comment]
Well if the dev team is looking into maybe a PP leaderboard and having combo removed from PP then that sort of solves your problem.

Seems like a thing to be fixed down the road though.

**Reply by PMorghK** [reply]
Well I dont really care too much about the pp. its just that the A score I got felt so much better compared to the top score. So im suggesting that maybe the scores should be diffrently wighted depending on the type of map.

Like in tournament play, stuff like the speed picks shouldnt be better for people that double tap it or combo game it. It should be whoever has the better acc because of the strict stamina and finger control requirements.

**Comment by bdach** [comment, marked as answer]
There is [an ongoing survey](https://osu.ppy.sh/home/news/2022-03-07-community-meetings-recap) that touches on general direction of score as a metric. Depending on the conclusion of that this problem may go away (eg. if pp sorting is considered preferable). If it doesn't, combo/accuracy proportions can be tweaked, but I'm not sure there is a ratio that will please everyone in all possible cases. You've brought up just one.

**Reply by PMorghK** [reply]
yea I guess thats pretty true. definitely wish it was at least a bit more dependent on the type and style of map.
